src/applications/education-letters/components/LetterResults.jsx

```diff
@@ -13,55 +13,44 @@ export const HasLetters = ({ claimStatus }) => {
 
   return (
     <>
-      <FormTitle title="Your VA education letter" />
+      <FormTitle title="Download your VA education decision letter" />
       <p className="va-introtext">
-        Download your VA education decision letter.
+        Check this page for your decision letter for Post-9/11 GI Bill benefits.
       </p>
-      <h2>Letter available for you to download</h2>
+      <h2>Your decision Letter</h2>
```

Comment:
Is capitalization ok here, @bradbergeron-us ?

Reply:
I pushed up the change. It should have been lowercase, according to the mock-up document.